SuperBuild/External_ITK.cmake

```diff
@@ -99,7 +99,7 @@ if(NOT DEFINED ITK_DIR AND NOT Slicer_USE_SYSTEM_${proj})
       -DBUILD_TESTING:BOOL=OFF
       -DBUILD_EXAMPLES:BOOL=OFF
       -DITK_LEGACY_REMOVE:BOOL=OFF
-      -DITKV4_COMPATIBILITY:BOOL=ON
+      -DITKV4_COMPATIBILITY:BOOL=OFF
```

Comment:
This will become more important soon, with the expected merging of this [patch](http://review.source.kitware.com/#/c/23819/1). You could leave it `ON`, or we could consider turning it `ON` once we update ITK to beta 2 or 5.0 final.